src/common/dlgcmn.cpp

```diff
@@ -196,19 +196,21 @@ wxDialogBase::GetParentForModalDialog(wxWindow *parent, long style) const
 
 #if wxUSE_STATTEXT
 
-wxSizer *wxDialogBase::CreateTextSizer(const wxString& message)
+wxSizer *wxDialogBase::CreateTextSizer(const wxString& message, int maxWidth)
 {
     wxTextSizerWrapper wrapper(this);
 
-    return CreateTextSizer(message, wrapper);
+    return CreateTextSizer(message, wrapper, maxWidth);
 }
 
 wxSizer *wxDialogBase::CreateTextSizer(const wxString& message,
-                                       wxTextSizerWrapper& wrapper)
+                                       wxTextSizerWrapper& wrapper,
+                                       int maxWidth)
 {
+    int widthMax = maxWidth;
```

Comment:
It's really confusing to have both `maxWidth` and `maxWidth` variables in one and the same function. Could you please either rename the argument to `widthMax` or the local variable to `maxWidth` and just remove the variable declaration/initialization in any case?